docs/shareseq/gex_atac.rst

```diff
@@ -0,0 +1,250 @@
+                    - "local" for local machine.
+                  - "gcp"
+                  - "gcp"
+		* - acronym_file
+		  - | The link/path of an index file in TSV format for fetching preset genome references, chemistry whitelists, etc. by their names.
+		    | Set an GS URI if *backend* is ``gcp``; an S3 URI for ``aws`` backend; an absolute file path for ``local`` backend.
+		  - "s3://xxxx/index.tsv"
+		  - "gs://xxxx/ref-data/shareseq/index.tsv"
+		* - config_version
+		  - Config docker version used for processing sample sheets
+		  - "0.2"
+		  - "0.2"
+		* - shareseq_mkfastq_version
+		  - SHARE-seq mkfastq version
+		  - "0.1.0"
+		  - "0.1.0"
+		* - shareseq_reorg_version
+		  - SHARE-seq reorg version
+		  - "0.1.0"
+		  - "0.1.0"
+		* - star_version
+		  - STAR version
+		  - "2.7.9a"
+		  - "2.7.9a"
+		* - chromap_version
+		  - Chromap version 
+		  - "0.1.4"
+		  - "0.1.4"
+		* - shareseq_mkfastq_num_cpu
+		  - Number of CPUs for shareseq_mkfastq
+		  - 32
+		  - 32
+		* - shareseq_mkfastq_memory
+		  - Memory shareseq_mkfastq.
+		  - "120G"
+		  - "120G"
+                * - sharseq_reorg_num_cpu
+                  - Number of CPUs for shareseq_reorg
+                  - 4
+                  - 4
+                * - sharseq_reorg_memory
+                  - Memory sharseq_reorg.
+                  - "8G"
+                  - "8G"
+                * - starsolo_num_cpu
+                  - Number of CPUs for STARsolo.
+                  - 32
+                  - 32
+                * - starsolo_memory
+                  - Memory for STARsolo.
+                  - "120G"
+                  - "120G"
+                * - chromap_num_cpu
+                  - Number of CPUs for Chromap.
+                  - 8
+                  - 8
+                * - chromap_memory
+                  - Memory for Chromap.
+                  - "64G"
+                  - "64G"
+                * - mkfastq_disk_space
+                  - Disk space for shareseq_mkfastq
+                  - 1500
+                  - 1500
+                * - shareseq_reorg_disk_space
+                  - Disk space for shareseq_reorg
+                  - 500
+                  - 500
+                * - starsolo_disk_space
+                  - Disk space for STARsolo
+                  - 500
+                  - 500
+                * - chromap_disk_space
+                  - Disk space for Chromap
+                  - 200
+                  - 200
+                * - preemptible
+                  - Number of preemptible tries
+                  - 2
+                  - 2
+                * - awsMaxRetries
+                  - Max number of retries for AWS instance
+                  - 5
+                  - 5
+
+Workflow output
++++++++++++++++
+
+See the table below for important SHARE-seq outputs.
+
+.. list-table::
+	:widths: 5 5 10
+	:header-rows: 1
+
+	* - Name
+	  - Type
+	  - Description
+	* - output_fastqs_directory
```

Comment:
Please revise this section based on the newly added output section of the shareseq master workflow. In addition, please describe in detail what's in each folder for gex_output and atac_output (e.g. for atac it is a bed file, what each column represents; for gex, it is OK to mention it is STARsolo outputs).